lib/Insulin.thy

```diff
@@ -55,7 +55,7 @@
 
 theory Insulin
 imports
-  Pure
+  Main
```

Comment:
Japheth probably put the Pure in with some kind of intent. Naturally, we don't do anything on top of Pure anyway, but any reason why it has to be Main now?

Reply:
Importing Pure is generally a bad idea, everything should be on HOL.Main, otherwise you can get weird context merge effects. Japheth wanted to express more generality, but we're not ever going to apply this to ZF or the lambda cube..